doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1260 @@
+
+To be copied from the Confluence page
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.5.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.5.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.5.0.
+
+:ref:`Toolchain Manager <gs_app_tcm>`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.5.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.5.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.5.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_250_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE and tool support
+====================
+
+|no_changes_yet_note|
+
+Application development
+=======================
+
+This section provides detailed lists of changes to overarching SDK systems and components.
+
+Build system
+------------
+
+* Removed the ``CONFIG_MCUBOOT_IMAGE_VERSION`` Kconfig option in favor of using a dedicated :ref:`application VERSION file <zephyr:app-version-details>` to set the version.
+  You can alternatively set the version by using the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` Kconfig option, but using a :file:`VERSION` file is the recommended approach.
+
+* The |NCS| name and version is now displayed instead of the Zephyr version as the default boot banner when applications boot.
+  This can be customized in user applications.
+
+nRF Front-End Modules
+---------------------
+
+* Updated the name of the ``nrf21540_ek`` shield to ``nrf21540ek``.
+
+Working with nRF91 Series
```

Comment:
Hi @peknis , regarding pan7 I found no good place to add this information. I am wondering if this can go to "working with nrf91 series" section. What do you think?

Reply:
Might be a good place for it.